CVLByExample/HookDisabledOnResetStorageCommand/README.md

```diff
@@ -0,0 +1,72 @@
+# Certora Example: Demonstrating Hooks and `reset_storage`
+
+## Overview
+
+This repository contains an example that demonstrates how hooks behave in CVL when the `reset_storage` command is used. The example uses a simple Solidity contract named `BankReset` and a CVL specification to show that hooks do not get triggered when storage is reset.
```

Comment:
need to give a small explanation of rest_storage as this is not even in the dcouemntation.

reset_storage restore to after constructor or just all zero? are ghost also restored? can we add more to this example? it can be a later task. will this pass if in the constructor the msg.sender is allocated funds? 

make sure to update the index page as well

Reply:
I'll add tickets for both Documentation and Example.

Reply:
https://certora.atlassian.net/browse/CERT-7088
https://certora.atlassian.net/browse/CERT-7089